Source/NVActivityIndicatorView/NVActivityIndicatorView.swift

```diff
@@ -49,6 +49,7 @@ import UIKit
  - BallScaleMultiple:       BallScaleMultiple animation.
  - BallPulseSync:           BallPulseSync animation.
  - BallBeat:                BallBeat animation.
+ - BallDoubleBounce:        BallDoubleBounce animation.
```

Comment:
Nitpick, `BallsBounce`.

Reply:
Naming is a thing so we can discuss a better one.

The inspiration one is called "double-bounce", i did this by duplicating an existing "ball" animation, so that`s why i tried "BallDoubleBounce".

In my opinion sounds better something like "PulsingBalls", but you chose :)